cinema/models.py

```diff
@@ -1,9 +1,35 @@
 from django.db import models
 
 
+class Actor(models.Model):
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return f"{self.first_name} {self.last_name}"
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+
+    def __str__(self):
+        return self.name
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.IntegerField()
+    seats_in_row = models.IntegerField()
+
+    def __str__(self):
+        return {self.name}
```

Comment:
There is a mistake in the `__str__` method of the `CinemaHall` class. The return statement should use parentheses for string formatting, not curly braces. It should be `return self.name` instead of `return {self.name}`.